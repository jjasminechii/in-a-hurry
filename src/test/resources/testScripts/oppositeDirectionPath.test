# Find a path with Dijkstra's algorithm when
# there is no path found for a complex graph

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B
AddNode simpleGraph C
AddNode simpleGraph D
AddNode simpleGraph E

AddEdge simpleGraph A B 3.0
AddEdge simpleGraph B C 1.0
AddEdge simpleGraph C D 1.0
AddEdge simpleGraph D E 3.0

FindPath simpleGraph E A