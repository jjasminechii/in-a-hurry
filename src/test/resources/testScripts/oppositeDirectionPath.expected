# Find a path with Dijkstra's algorithm when
# there is no path found for a complex graph

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph
added node C to simpleGraph
added node D to simpleGraph
added node E to simpleGraph

added edge 3.000 from A to B in simpleGraph
added edge 1.000 from B to C in simpleGraph
added edge 1.000 from C to D in simpleGraph
added edge 3.000 from D to E in simpleGraph

path from E to A:
no path found